### PR TITLE
Update ChangeLog and NEWS for 10.45

### PR DESCRIPTION
**Pull request by NWilson**

### CODE REVIEW

**Review by zherczeg — APPROVED** [review]
LGTM! Thank you for collecting all data.

File: ChangeLog [hunk 1]
```diff
@@ -47,77 +54,144 @@ changed as follows:
...
-20. Merged PR470, which adds PCRE2_EXTRA_NO_BS0 and PCRE2_EXTRA_PYTHON_OCTAL.
+31. (#470) Add option flags PCRE2_EXTRA_NO_BS0 and PCRE2_EXTRA_PYTHON_OCTAL.
 
-21. Prevent 1 byte overread when parsing malformed patterns with early VERBs.
+32. (#471) Add new API function pcre2_set_optimize() for controlling which
+optimizations are enabled.
 
-22. Merged PR491 which adds $& $` $' and $_ to substitution replacements, as
-well as interpreting \b and \v as characters.
+33. (#487) Prevent 1 byte overread when parsing malformed patterns with early
```
[the remark this answers is not in the document]
Comment (NWilson) [reply ChangeLog:137]: Oops! I didn't realise they were the same. I'll merge.

**Comment by zherczeg** [comment]
Does it mean we do the release? Today?

**Comment by NWilson** [comment]
No! I haven't released anything, just updated the ChangeLog so we're ready.

I won't release until you're ready.

**Comment by zherczeg** [comment]
I am ready, so you can do the release anytime. My little projects are not urgent, they can wait. I will focus on bugfixes after the release.

**Comment by NWilson** [comment]
@zherczeg Let's do this then! It's only an RC, we just want to gather some feedback.

I've made a _Draft_ release here: https://github.com/PCRE2Project/pcre2/releases

Everything seems  OK, to me...

**Comment by zherczeg** [comment]
We should announce the RC1 in pcre2-dev:
https://groups.google.com/g/pcre2-dev?pli=1

I think this is a managerial task, but I (or maybe Philip if he wants) can do it if you prefer that as well.

**Comment by zherczeg** [comment]
Btw the other RC-s has "Pre-release" tag, and this one has "Draft", and emojies cannot be used. Is it visible to everybody?

**Comment by NWilson** [comment]
It was not visible. I have just published it! 🥳

**Pull request merged**